modules/configure-ocp-oauth-identity-provider.adoc

```diff
@@ -22,5 +22,28 @@ To integrate the built-in {ocp} OAuth server as an identity provider for {produc
 ====
 For security, Red Hat recommends setting the *Minimum access role* to *None* while you complete setup. Later, you can return to the *Access Control* page to set up more tailored access rules based on user metadata from your identity provider.
 ====
-. To add access rules for users and groups accessing {product-title-short}, use the *Rules* section. For example, to give the *Admin* role to a user called `administrator`, select *User* as *Key*, *administrator* as *Value*, and *Admin* as the *Role*. Use *Add new rule* to add more rules.
+
+. To add access rules for users and groups accessing {product-title-short}, use the *Rules* section. For example:
+.. To give the *Admin* role to a user called `administrator`, you can use the following key-value pairs to create access rules:
++
+[cols="1,1"]
+|===
+|Key |Value
```

Comment:
Do you want to highlight **Key** and **Value** to distinguish them, from the other rows and columns?